install.sh

```diff
@@ -995,18 +995,23 @@ case "${SHELL}" in
 esac
 
 # show different instructions if the user already has Homebrew in their PATH
-if grep -qs "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}"
+if grep -qs "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}" && ! type brew >/dev/null 2>/dev/null
 then
 cat <<EOS
 - Homebrew was already found in your PATH, run this command to activate it in your current shell:
     eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"
 EOS
-else
+elif ! grep -q "eval \"\$(${HOMEBREW_PREFIX}/bin/brew shellenv)\"" "${shell_profile}"
+then
 cat <<EOS
 - Run these two commands in your terminal to add Homebrew to your ${tty_bold}PATH${tty_reset}:
     (echo; echo 'eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"') >> ${shell_profile}
     eval "\$(${HOMEBREW_PREFIX}/bin/brew shellenv)"
 EOS
+else
+cat <<EOS
+- Homebrew is already set up. Your system is ready to brew.
```

Comment:
Is there any chance more steps will be output below? If so, we should probably not do this.

Reply:
No, more steps will not display in that case.

Reply:
What about the `Run these commands in your terminal`, `Install Homebrew's dependencies`, etc. below?

Reply:
My bad for not seeing that. I will change the wording.

Reply:
> If so, we should probably not do this.

By not doing this, I infer you mean remove the prompt in the case Homebrew is set up. Correct?

Reply:
Display nothing in this case, I think, sorry!